sources/api/apiclient/Cargo.toml

```diff
@@ -23,7 +23,7 @@ futures.workspace = true
 futures-channel.workspace = true
 http.workspace = true
 httparse.workspace = true
-hyper = { workspace = true, features = ["client", "http1", "http2", "tcp"] }
+hyper = { version = "0.14", features = ["client", "http1", "http2", "tcp"] }
```

Comment:
should we add a comment here as to why we're pinning `hyper` to a different version for apiclient?

Reply:
I think we want to stay (at least for now) on hyper 0.14 for `apiclient` because we use it for the `apiclient exec` feature and we don't want to migrate to a newer hyper version for the FIPS support.